### PR TITLE
introduce about dialog

### PR DESCRIPTION
**Pull request by ElderOrb**

### CODE REVIEW

**Comment by ElderOrb** [comment]
https://github.com/mipops/dvrescue/issues/502

File: Source/GUI/dvrescue/dvrescue/AboutDialog.qml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            Layout.alignment: Qt.AlignHCenter
+        }
+
+        Text {
+            text: "Build Qt version: " + buildQtVersion + ", runtime Qt version: " + runtimeQtVersion
+            Layout.alignment: Qt.AlignHCenter
+        }
+
+        Text {
+            text: "Build FFMpeg version: " + buildFFMpegVersion + ", runtime FFMpeg version: " + runtimeFFMpegVersion
```
Comment (dericed) [inline Source/GUI/dvrescue/dvrescue/AboutDialog.qml:71]: please change to `FFmpeg` in both quotes

File: Source/GUI/dvrescue/dvrescue/main.qml [hunk 2]
```diff
@@ -81,6 +92,14 @@ ApplicationWindow {
             }
             icon.source: "/icons/menu-debug.svg"
         }
+
+        NavButton {
+            // text: qsTr("Debug")
+            onClicked: {
+                about.open()
+            }
+            icon.source: "/icons/menu-debug.svg"
```
Comment (dericed) [inline Source/GUI/dvrescue/dvrescue/main.qml:101]: I see the page is accessed via a copy/paste of the "DEBUG" button. I suggest removing it from the sidecar altogether and instead have it in the toolbar, such as in "dvrescue > About dvrescue", similar to how it's accessed in QCTools.

File: Source/GUI/dvrescue/dvrescue/AboutDialog.qml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+    ColumnLayout {
+        id: layout
+        Image {
+            source: "/dvrescue.png"
+            Layout.alignment: Qt.AlignHCenter
+        }
+
+        Text {
+            text: "DVRescue"
```
Comment (dericed) [inline Source/GUI/dvrescue/dvrescue/AboutDialog.qml:32]: I think this text can be dropped, it's redundant to the above image.

File: Source/GUI/dvrescue/dvrescue/AboutDialog.qml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            }
+
+            Text {
+                text: version
+                font.pixelSize: 20
+            }
+        }
+
+        Text {
+            text: buildDate
```
Comment (dericed) [inline Source/GUI/dvrescue/dvrescue/AboutDialog.qml:51]: can you move the buildDate to the end of the above line. Such as `Version $Version - $buildDate`

File: Source/GUI/dvrescue/dvrescue/AboutDialog.qml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+            Layout.alignment: Qt.AlignHCenter
+        }
+
+        Text {
+            text: "Copyright (c) 2019-2022, Moving Image Preservation of Puget Sound."
```
Comment (dericed) [inline Source/GUI/dvrescue/dvrescue/AboutDialog.qml:56]: Here are elsewhere, how about the `©` symbol?

Comment (dericed) [reply Source/GUI/dvrescue/dvrescue/AboutDialog.qml:56]: hyperlink the org title to https://www.mipops.org/

File: Source/GUI/dvrescue/dvrescue/AboutDialog.qml [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+            Layout.alignment: Qt.AlignHCenter
+        }
+
+        Text {
+            text: "Third party libraries Copyright (c) 2012-2020 MediaArea.net SARL"
```
Comment (dericed) [inline Source/GUI/dvrescue/dvrescue/AboutDialog.qml:61]: hyperlink org title to https://mediaarea.net/

File: Source/GUI/dvrescue/dvrescue/AboutDialog.qml (hunk 1 above)
Comment (dericed) [reply Source/GUI/dvrescue/dvrescue/AboutDialog.qml:71]: Update FFmpeg text to comply with parts 9 and 10 of https://ffmpeg.org/legal.html.

**Review by dericed — APPROVED** [review]
This looks good, I left a few comments include on the case of "FFmpeg" and about how to access the about page. Also with the modal window it isn't clear right away how to leave (I found escape works) can you add a Close button or 'x'.

**Comment by dericed** [comment]
Can you add `This software uses libraries from the FFmpeg project under the LGPLv2.1` as a line before the line that lists the ffmpeg versions.

Also, the link to the About page looks like: 

image

but can you make it in the toolbar menu like:

image

**Pull request merged**